specify/specify7
# Prompt the user with interaction dialog when adding a new prep

Fixes #4523

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

Go to new Gift
Click on "Without Preparations"
Scroll down to Gift Preparations and click the +
See that the interaction dialog opens
Use different options to add new prep 
Do the same for loans

Can you check if it fixe #4078
